reinforce/dqn/agent.py

```
@@ -0,0 +1,367 @@
+            rewards, next_q_dist, dones, self.gamma, self.n_step, self.v_min, self.v_max, self.num_atoms
+        )
+
+        # ##>: Compute loss and update.
+        with tf.GradientTape() as tape:
+            q_dist = self.online_network(states, training=True)
+            loss, td_errors = categorical_dqn_loss(q_dist, target_dist, actions, weights)
+
+        gradients = tape.gradient(loss, self.online_network.trainable_variables)
+        # ##>: Clip gradients.
+        gradients = [tf.clip_by_norm(g, 10.0) if g is not None else g for g in gradients]
+        self.optimizer.apply_gradients(zip(gradients, self.online_network.trainable_variables))
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)